### PR TITLE
Add md exclude tags for spouse and recsys

### PR DESCRIPTION
**Pull request by brahmaneya**
**Test Plan**
tox. Also created markdowns and viewed in local website.

### CODE REVIEW

File: recsys/recsys_tutorial.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
 # We look at a sample of the labeled development set.
 # As an example, we want our final recommendations model to be able to predict that a user who has interacted with `book_idxs` (25743, 22318, 7662, 6857, 83, 14495, 30664, ...) would either not read or not like the book with `book_idx` 22764 (first row), while a user who has interacted with `book_idxs` (3880, 18078, 9092, 29933, 1511, 8560, ...) would read and like the book with `book_idx` 3181 (second row).
 
-# %%
+# %% {"tags": ["md-exclude"]}
 df_dev.sample(frac=1, random_state=12).head()
```
Comment (bhancock8) [inline recsys/recsys_tutorial.py]: Fine to exclude this output, but this line needs a comment explaining what's happening regardless.

File: recsys/recsys_tutorial.py [hunk 2]
```diff
@@ -180,7 +182,7 @@ def polarity_negative(x):
     return ABSTAIN
 
 
-# %%
+# %% {"tags": ["md-exclude-output"]}
```
Comment (bhancock8) [inline recsys/recsys_tutorial.py]: We don't need to hide this anymore with the better horz scrolling

File: recsys/recsys_tutorial.py [hunk 3]
```diff
@@ -99,7 +99,9 @@ def shared_first_author(x, book_to_first_author, first_author_to_books):
 
 # %% [markdown]
 # We can also leverage the long text reviews written by users to guess whether they liked or disliked a book.
+# %% [markdown] {"tags": ["md-exclude"]}
```
Comment (bhancock8) [inline recsys/recsys_tutorial.py]: This is okay now that we have dataframe scrolling

File: recsys/recsys_tutorial.py [hunk 4]
```diff
@@ -343,11 +345,11 @@ def get_data_tensors(df):
 # %% [markdown]
 # Finally, we evaluate the model's predicted ratings on our test data.
 #
-# %%
+# %% {"tags": ["md-exclude-output"]}
```
Comment (bhancock8) [inline recsys/recsys_tutorial.py]: Do we not get to the see the final score? I think that's worth reporting.

File: spouse/spouse_demo.py [hunk 5]
```diff
@@ -248,20 +249,21 @@ def lf_distant_supervision_last_names(x, last_names):
...
 applier = PandasLFApplier(lfs)
 
-# %%
+# %% {"tags": ["md-exclude-output"]}
 from snorkel.labeling import LFAnalysis
 
 L_dev = applier.apply(df_dev)
 L_train = applier.apply(df_train)
 
+# %% {"tags": ["md-exclude"]}
```
Comment (bhancock8) [inline spouse/spouse_demo.py]: OK now

File: recsys/recsys_tutorial.py (hunk 1 above)
Comment (brahmaneya) [reply recsys/recsys_tutorial.py]: I'm excluding the whole cell, not just the output.

**Comment by brahmaneya** [comment]
Current travis run will fail because I didn't sync notebook. I'll do a final sync once the py file content is finalized.

**Pull request merged**